tICA/README.md

```diff
@@ -1,22 +1,23 @@
 # Temporal ICA
 
-This pipeline uses ICA on a large amount of fMRI data (typically a large group of subjects),
+This pipeline uses [temporal ICA][tICA] on a large amount of fMRI data (typically a large group of subjects),
 in order to separate global nuisance effects (such as due to respiration) from neural signals that
 have a nonzero spatial mean.
 
 The processing modes also allow for applying temporal ICA cleanup to datasets that are not
 large enough to reliably produce a new temporal ICA decomposition.  Briefly, the processing
 modes are:
 
-- NEW - do a full pipeline run from scratch, starting with a fresh [MIGP], intended for large datasets
-- REUSE_SICA_ONLY - use a previous MIGP result, but estimate the temporal ICA from scratch ([ICASSO] with random seeds)
-- INITIALIZE_TICA - use a previous MIGP result, and estimate temporal ICA starting from a previous
+- NEW - do a full pipeline run from scratch, starting with a fresh group-PCA (using [MIGP]), intended for large datasets
+- REUSE_SICA_ONLY - use a previous group-PCA result, but estimate the temporal ICA from scratch ([ICASSO] with random seeds)
+- INITIALIZE_TICA - use a previous group-PCA result, and estimate temporal ICA starting from a previous
 tICA mixing matrix, but allow it to change to fit the new data (ICASSO starting from prior result)
-- REUSE_TICA - use a previous MIGP and tICA mixing matrix (and signal/nuisance classification),
+- REUSE_TICA - use a previous group-PCA and tICA mixing matrix (and signal/nuisance classification),
 intended for small datasets
 
 Automated classification of tICA components into signal versus nuisance is not yet integrated.
```

Comment:
Can we elaborate on the implications of this for how users would actually run tICA cleanup _at this time_?

Reply:
Referring to the "not yet integrated" line.

Reply:
I believe we developed some internal code to help with manual classification, in addition to the not integrated automated classification code, but we may not want to try to integrate them now.  The current tICA pipeline code can be useful for ordinary users to clean their data based on one of our prior runs.

Reply:
I'm not saying that we need the automated classification at this time.  I *do* think we need examples of how people are actually supposed to run the tICA-cleanup within the context of its current capabilities.

Reply:
Personally, when I see "Automated classification is not yet integrated", I think "okay, so the options are manual or using an existing classification", and only one mode mentions a previous classification (or even makes sense to use with a previous classification), so adding more words that just say that seems a bit redundant to me.

I expect the plan is to provide additional instructions once we have released a tICA dataset that people can use with REUSE_TICA mode, but since at present we haven't published such a dataset, writing our best guess at those instructions here would be premature.

Reply:
@glasserm is there anything we can say here to help a user that is willing to do a manual classification?

Reply:
I would say this is alpha or beta as far as documentation.  This will need to be improved with further publications.  We need to get this out so that processing can proceed.

Reply:
I have updated the readme to indicate beta status.  @mharms, do you have any major concerns remaining, or concerns other than documentation?